tonic/models/vic/grid_params.py

```diff
@@ -34,6 +35,32 @@
 NC_CHAR = 'S1'
 MAX_NC_CHARS = 256
 
+# option values
+FROM_DEFAULT = 0
+FROM_VEGLIB = 1
+FROM_VEGPARAM = 2
```

Comment:
In Python string comparison is pretty trivial (`if x == 'FROM_DEFAULT'`).  I'd rather just compare the strings rather than alias to `int`s.